imswitch/imcontrol/controller/controllers/AufofocusController.py

```diff
@@ -52,6 +49,12 @@ def __init__(self, *args, **kwargs):
         self.imageToDisplayName = ""
         self.imageToDisplay = None
         self.sigImageReceived.connect(self.displayImage)
+        
+        # Connect AutofocusWidget buttons
+        if not imswitch.IS_HEADLESS: # TODO: We need to have signals instead!
```

Comment:
instead of importing the whole `imswitch` module import only the `IS_HEADLESS` flag

Reply:
I gave it some tought and I think it doesn't necesseraly impact performance so we can keep it like this.